launchpod/src/test/java/com/google/launchpod/DisplayFeedServletTest.java

```diff
@@ -0,0 +1,176 @@
+import javax.servlet.http.HttpServletResponse;
+import com.google.launchpod.servlets.DisplayFeedServlet;
+import com.google.launchpod.data.UserFeed;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+/** 
+ * Runs unit tests for the DisplayFeedServlet that contains a doGet() method.
+*/
+@RunWith(JUnit4.class)
+public class DisplayFeedServletTest extends Mockito {
+
+    @InjectMocks
+    private DisplayFeedServlet servlet = new DisplayFeedServlet();
+
+    @Mock
+    HttpServletRequest request;
+
+    @Mock
+    HttpServletResponse response;
+    
+    @Rule // JUnit 4 uses Rules for testing specific messages
+    public ExpectedException thrown = ExpectedException.none();
+
+    private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig()
+    .setDefaultHighRepJobPolicyUnappliedJobPercentage(100));
+
+    // keys
+    private static final String USER_FEED = "UserFeed";
+    private static final String PODCAST_TITLE = "title";
+    private static final String XML_STRING = "xmlString";
+    private static final String MP3_LINK = "mp3link"; // URL to existing MP3 file
+    private static final String TIMESTAMP = "timestamp";
+    private static final String EMAIL = "email"; 
+    private static final String ID = "id";
+
+    private static final String TEST_PODCAST_TITLE = "TEST_PODCAST_TITLE";
+    private static final String TEST_EMPTY_PODCAST_TITLE = "";
+    private static final String TEST_NULL_PODCAST_TITLE = null;
+    private static final String TEST_MP3_LINK = "TEST_MP3_LINK";
+    private static final String TEST_EMPTY_MP3_LINK = "";
+    private static final String TEST_NULL_MP3_LINK = null;
+    private static final long TEST_TIMESTAMP = System.currentTimeMillis();
+    private static final String TEST_ID = "123456";
+    private static final String TEST_ID_TWO = "789012";
+    private static final String TEST_XML_STRING = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"  +
+                       "<rss version=\"2.0\">" + 
+                       "  <channel>" +
+                       "    <language>en</language>"  +
+                       "    <itunes:author>User</itunes:author>" + 
+                       "    <title>" + TEST_PODCAST_TITLE + "</title>" + 
+                       "    <item>" +
+                       "      <title>" + TEST_PODCAST_TITLE + "</title>" + 
```

Comment:
I think we just put TEST_PODCAST_TITLE as the title for both the podcast and the episode since for the prototype we're not really asking for the specific episode. But yes @ERetana2 for MVP if we can we should replace the hard-coded "This is episode 4" with an episode number